content/en/data_observability/integrations/openlineage/_index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: OpenLineage
+description: "Connect OpenLineage to Datadog to capture lineage from custom systems and orchestrators."
+further_reading:
+  - link: '/data_observability/'
+    tag: 'Documentation'
+    text: 'Data Observability'
+  - link: '/data_observability/integrations/openlineage/datadog_agent_for_openlineage'
+    tag: 'Documentation'
+    text: 'Set up Datadog Agent for OpenLineage Proxy'
+---
+
+## Overview
+
+Lineage in Datadog Data Observability is built on [OpenLineage][1], the open source standard for collecting and analyzing data lineage. OpenLineage is especially helpful when you need to:
+
+- Capture lineage from systems Datadog doesn't integrate with natively, such as in-house tools
+- Emit lineage events for jobs or orchestrators where building a native Datadog integration would be difficult
+
+Replace the hostname in the examples with the relevant [Datadog site][2] for your organization. You can find your Datadog site by [following these instructions][3]. This example uses `datadoghq.com`.
```

Comment:
```suggestion
Replace the hostname in the examples with the relevant [Datadog site][2] for your organization. To find your Datadog site, see [Access the Datadog site][3]. This example uses `datadoghq.com`.
```
Again, not a huge thing by any means, there's just a lot of links floating around and it can help to give some context about whether one stays in the docs or goes somewhere else!